docker/start.sh

```
@@ -0,0 +1,94 @@
+export VSO_AGENT_IGNORE="AZP_TOKEN,AZP_TOKEN_FILE"
+
+print_header "1. Determining matching Azure Pipelines agent..."
+
+AZP_AGENT_PACKAGES=$(curl -LsS \
+    -u user:$(cat "${AZP_TOKEN_FILE}") \
+    -H "Accept:application/json" \
+    "${AZP_URL}/_apis/distributedtask/packages/agent?platform=${TARGETARCH}&top=1")
+
+AZP_AGENT_PACKAGE_LATEST_URL=$(echo "${AZP_AGENT_PACKAGES}" | jq -r ".value[0].downloadUrl")
+
+if [ -z "${AZP_AGENT_PACKAGE_LATEST_URL}" -o "${AZP_AGENT_PACKAGE_LATEST_URL}" == "null" ]; then
```

Shellcheck (0.10.0)

[warning] Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.

(SC2166)